metrics/grafana/tiflash_summary.json

```diff
@@ -4186,6 +4186,22 @@
               "intervalFactor": 1,
               "legendFormat": "AIO",
               "refId": "F"
+            },
+            {
+              "expr": "sum(rate(tiflash_system_profile_event_PSV3MBlobExpansion{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m]) + rate(tiflash_system_profile_event_PSV3MBlobReused{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m])) by (instance)",
+              "format": "time_series",
+              "intervalFactor": 1,
+              "hide": true,
+              "legendFormat": "BlobQPS",
+              "refId": "G"
+            },
+            {
+              "expr": "sum(rate(tiflash_system_profile_event_PSV3MBlobReused{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m]) /  (rate(tiflash_system_profile_event_PSV3MBlobExpansion{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m]) + 1)) by (instance)",
+              "format": "time_series",
+              "intervalFactor": 1,
+              "hide": true,
+              "legendFormat": "BlobReusedRate",
+              "refId": "H"
             }
```

Comment:
This is a rate between [0.0, 1.0], showing it in the "Disk Write OPS" panel seems weird

Maybe use a separate panel for "BlobQPS" and "BlobReusedRate"?

Reply:
It's actually not between [0.0, 1.0], cause it's `reused_ratio = BlobReused / BlobExpansion`. 

why not used `BlobReused / QPS`?
1. I do want to keep this rate into pannel， instead of adding a new panel.
2. Although they look different, they actually mean the same thing.

If we do added a `reused_ratio` pannel, But only one piece of data can be placed in it, `BlobQPS` won't add into this new pannel. Also a little confusing?

Reply:
Without "BlobQPS" and "BlobReusedRate", summing the values in "Disk Write OPS" is almost the same write ops to the disk. This can help us to figure out whether the IOPS to the disk is high or not.

"BlobQPS" does not really means a "write operation" to the disk, it only effect the memory data structure "BlobStat". The "write operation" is shown as "PageFile". That's the main reason that I think "BlobQPS" and "BlobReusedRate" should not belong to this panel.

Reply:
added into new pannel, pic below